cmd/cosign/cli/sign/sign.go

```diff
@@ -155,7 +152,7 @@ func SignCmd(ro *options.RootOptions, ko options.KeyOpts, signOpts options.SignO
 
 	var staticPayload []byte
 	if signOpts.PayloadPath != "" {
-		ui.Info(ctx, "Using payload from:", signOpts.PayloadPath)
+		ui.Info(ctx, "Using payload from: %s", signOpts.PayloadPath)
```

Comment:
Nit will be good to have this called InfoF to match similar pattern like PrintF SPrintF ...

Reply:
Done